test/Azure.Functions.Cli.Tests/E2E/CreateFunctionTests.cs

```diff
@@ -214,7 +214,7 @@ await CliTester.Run(new RunConfiguration
             }, _output);
         }
 
-        [Fact]
+        [Fact(Skip="Flaky test")]
```

Comment:
Can we please create an issue to track all skipped tests in this PR?

Reply:
Added https://github.com/Azure/azure-functions-core-tools/issues/3828